src/main/webapp/gast/servlet/header.jsp

```diff
@@ -1,33 +1,61 @@
 <%@page contentType="text/html" pageEncoding="UTF-8"%>
 <!DOCTYPE html>
-<HTML>
-  <HEAD>
+<html lang="###LANGUAGE_CODE###" class="no-js mod_picture mod_localstorage mod_cssremunit mod_placeholder mod_srcset mod_mediaqueries mod_no-touchevents mod_formvalidation mod_flexbox mod_csstransitions js-state__ut-nav-is-closed">
```

Comment:
`###LANGUAGE_CODE###` is a placeholder that should be replaced dynamically with the selected UI language.

Reply:
later in another pr when language will be done